flux-middle/src/rty/mod.rs

```diff
@@ -84,14 +85,15 @@ pub enum Sort {
     Int,
     Bool,
     Real,
+    BitVec(usize),
     Loc,
     Param(ParamTy),
     Tuple(List<Sort>),
     Func(FuncSort),
     User(Symbol),
 }
 
-#[derive(Clone, PartialEq, Eq, Hash, TyEncodable, TyDecodable)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, TyEncodable, TyDecodable)]
```

Comment:
You can add `FuncSort` to the macro `impl_debug_with_default_cx` at the end of this file to get a pretty printed version of `Debug` (I think `Pretty` is implemented for `FuncSort`)